tools/socketgen/socketgen.py

```diff
@@ -648,6 +659,41 @@ def write_acc_port_map(f, acc, dma_width, datatype, rst, is_noc_interface, is_vi
     f.write("      dma_write_chnl_msg         => dma_write_chnl_data,\n")
     f.write("      acc_done                   => acc_done\n")
     f.write("    );\n")
+
+    # f.write("    port map(\n")
+    # conf_info_size = 0
+    # for param in acc.param:
+    #   if not param.readonly:
+    #       conf_info_size += param.size
+    # for param in acc.param:
+    #   if not param.readonly:
+    #     spacing = " "
+    #     if 16 - len(param.name) > 0:
+    #       spacing = (16-len(param.name))*" "
+    #     f.write("      conf_info(" + str(conf_info_size-1) + " downto " + str(conf_info_size-param.size) + ") => " + "conf_info_" + param.name +",\n")
+    #     conf_info_size -= param.size
+    # f.write("      clk                        => clk,\n")
+    # spacing = (27-len(rst))*" "
+    # f.write("      " + rst + spacing       + "=> acc_rst,\n")
+    # f.write("      conf_done                  => conf_done,\n")
+    # f.write("      dma_read_ctrl_val          => dma_read_ctrl_valid,\n")
+    # f.write("      dma_read_ctrl_rdy          => dma_read_ctrl_ready,\n")
+    # f.write("      dma_read_ctrl_msg(" + str(66) + " downto " + str(64) + ") => dma_read_ctrl_data_size,\n")
+    # f.write("      dma_read_ctrl_msg(" + str(63) + " downto " + str(32) + ") => dma_read_ctrl_data_length,\n")
+    # f.write("      dma_read_ctrl_msg(" + str(31) + " downto " + str(0) + ") => dma_read_ctrl_data_index,\n")
+    # f.write("      dma_write_ctrl_val         => dma_write_ctrl_valid,\n")
+    # f.write("      dma_write_ctrl_rdy         => dma_write_ctrl_ready,\n")
+    # f.write("      dma_write_ctrl_msg(" + str(66) + " downto " + str(64) + ") => dma_write_ctrl_data_size,\n")
+    # f.write("      dma_write_ctrl_msg(" + str(63) + " downto " + str(32) + ") => dma_write_ctrl_data_length,\n")
+    # f.write("      dma_write_ctrl_msg(" + str(31) + " downto " + str(0) + ") => dma_write_ctrl_data_index,\n")
+    # f.write("      dma_read_chnl_val          => dma_read_chnl_valid,\n")
+    # f.write("      dma_read_chnl_rdy          => dma_read_chnl_ready,\n")
+    # f.write("      dma_read_chnl_msg          => dma_read_chnl_data,\n")
+    # f.write("      dma_write_chnl_val         => dma_write_chnl_valid,\n")
+    # f.write("      dma_write_chnl_rdy         => dma_write_chnl_ready,\n")
+    # f.write("      dma_write_chnl_msg         => dma_write_chnl_data,\n")
+    # f.write("      acc_done                   => acc_done\n")
+    # f.write("    );\n")
   else:
     f.write("    port map(\n")
```

Comment:
why do you have changes inside this "else" branch? wouldn't that not be for catapult_sysc?

Reply:
I don't see any change in the final "else" branch. What are you referring to ?